lib/livebook/teams/connection.ex

```diff
@@ -67,9 +71,11 @@ defmodule Livebook.Teams.Connection do
     case WebSocket.send(data.http_conn, data.websocket, data.ref, :ping) do
       {:ok, conn, websocket} ->
         Process.send_after(self(), {:loop_ping, data.ref}, @loop_ping_delay)
+        Logger.warning("Teams WebSocket connection - ping with success")
         {:keep_state, %{data | http_conn: conn, websocket: websocket}}
 
       {:error, conn, websocket, _reason} ->
+        Logger.warning("Teams WebSocket connection - ping with error")
```

Comment:
```suggestion
        Logger.warning("Teams WebSocket connection - ping error")
```

Reply:
I wonder if we are entering an error state where we ping and it always errors? Although I would expect the ping error to eventually become a disconnection.